core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadOnlyLDAPUserStoreManager.java

```diff
@@ -2917,23 +2918,41 @@ private void getExpressionConditionsAsList(Condition condition, List<ExpressionC
         return users;
     }
 
+    /**
+     * Method to verify whether the filter operation is a single attribute filter scenario or multi attribute filter scenario
```

Comment:
comment exceeds the maximum width

Reply:
Fixed: https://github.com/wso2/carbon-kernel/pull/2486